index.html

```diff
@@ -226,6 +226,28 @@ <h4>Skills</h4>
       </div>
     </section>
 
-  </body>
+    <footer class="flex footer">
+      <div class="flex footer-wrapper">
+        <h2 class="section-title">Contact me</h2>
+        <p class="section-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+        <form action="https://formspree.io/f/mgedpajj" method="post" class="form-wrapper">
+          <fieldset class="form-fieldset">
+            <div class="form-group">
+              <input type="text" name="name" id="name" class="form-input" placeholder="Your name" required maxlength="30">
+            </div>
+            <div class="form-group">
+              <input type="email" name="email" id="email" class="form-input" placeholder="Your email" required>
+            </div>
+            <div class="form-group">
+              <textarea name="message" rows="10" cols="30" id="message" class="form-input" placeholder="Please write your message here" maxlength="500" required></textarea>
+            </div>
+            <div class="form-group">
+              <button type="submit" class="button button-enabled contact-button">Get In Touch</button>
+            </div>
+          </fieldset>
```

Comment:
- [ ] Please give your fieldset some margin-right like margin-right: 15px; to make the form in the center.

![Screenshot from 2022-04-26 22-01-51](https://user-images.githubusercontent.com/33752735/165382610-fba2a6e7-2342-44ff-a5f1-b67cddc2a0dd.png)

![Screenshot from 2022-04-26 22-02-22](https://user-images.githubusercontent.com/33752735/165382653-d684b9da-a76c-4114-8a79-bb79de068c31.png)